lib/protocol/compressor.js

```diff
@@ -242,7 +242,7 @@ HeaderSetDecompressor.prototype._execute = function _execute(rep) {
   var entry, pair;
 
   if (rep.contextUpdate) {
-    this.setTableSizeLimit(rep.newMaxSize);
+   this._table.setSizeLimit(rep.newMaxSize);
```

Comment:
nit: that second space you deleted at the beginning of this line shouldn't have been deleted :)